zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -79,12 +79,8 @@ angular.module('zeppelinWebApp')
   var angularObjectRegistry = {};
 
   var editorModes = {
-    'ace/mode/python': /^%(\w*\.)?(pyspark|python)\s*$/,
-    'ace/mode/scala': /^%(\w*\.)?spark\s*$/,
-    'ace/mode/r': /^%(\w*\.)?(r|sparkr|knitr)\s*$/,
     'ace/mode/sql': /^%(\w*\.)?\wql/,
-    'ace/mode/markdown': /^%md/,
-    'ace/mode/sh': /^%sh/
+    'ace/mode/markdown': /^%md/
```

Comment:
Not sure how this would work with `jdbc`'s alias magic style too, eg. `%jdbc(hive)`, or `%hive`?